text_generation/causal_lm/cpp/set_up_and_run.sh

```diff
@@ -23,4 +23,4 @@ cmake --build ./build/ --config Release -j
 wait
 
 python ./convert_tokenizers.py ./open_llama_3b_v2/
-./build/casual_lm ./open_llama_3b_v2/openvino_model.xml ./tokenizer.xml ./detokenizer.xml "return 0"
+./build/casual_lm ./open_llama_3b_v2/openvino_model.xml ./tokenizer.xml ./detokenizer.xml "Why is the Sun yellow?"
```

Comment:
I'm fine if the test will pas. But as far as I remember that prompt results in infenit inference because EOS token is never produced.

Reply:
Thanks, I will checkout locally first. 
May I know if this script is used only for CI check?

Reply:
Yes, only for CI. The idea was that people could reuse it. But later I extended tests with timeout: https://github.com/openvinotoolkit/openvino.genai/pull/38 and it became impossible to reuse it for group beam search

Reply:
@Wovchena, I revert this change first to make CI green. 
BTW, do you think it is reasonable to add additional argument "max_nex_token" for loop condition: https://github.com/openvinotoolkit/openvino.genai/blob/0fae1f4c0fbe37dab68f35edcfbe4a1d0ac35742/text_generation/casual_lm/cpp/casual_lm.cpp#L72,  and this arguments can be pass by cmd line, which can better handle timeout cause by "EOS token never produced" issue.

Reply:
> EOS token never produced" issue

is a CI issue and there's a way to solve it - use different prompt